packages/grafana-ui/src/components/Table/TableNG/utils.ts

```diff
@@ -510,23 +508,16 @@ export function getCellColors(
     if (mode === TableCellBackgroundDisplayMode.Basic) {
       textColor = getTextColorForAlphaBackground(displayValue.color!, theme.isDark);
       bgColor = tinycolor(displayValue.color).toRgbString();
-      // bgHoverColor = tinycolor(displayValue.color)
-      //   .darken(CELL_COLOR_DARKENING_MULTIPLIER * darkeningFactor)
-      //   .toRgbString();
     } else if (mode === TableCellBackgroundDisplayMode.Gradient) {
-      // const hoverColor = tinycolor(displayValue.color)
-      //   .darken(CELL_GRADIENT_DARKENING_MULTIPLIER * darkeningFactor)
-      //   .toRgbString();
       const bgColor2 = tinycolor(displayValue.color)
         .darken(CELL_COLOR_DARKENING_MULTIPLIER * darkeningFactor)
         .spin(CELL_GRADIENT_HUE_ROTATION_DEGREES);
       textColor = getTextColorForAlphaBackground(displayValue.color!, theme.isDark);
       bgColor = `linear-gradient(120deg, ${bgColor2.toRgbString()}, ${displayValue.color})`;
-      // bgHoverColor = `linear-gradient(120deg, ${bgColor2.toRgbString()}, ${hoverColor})`;
     }
   }
 
-  return { textColor, bgColor };
+  return { color: textColor, background: bgColor };
```

Comment:
one thing I opted to do based on feedback that there was too much duplicate logic for tooltip rendering and cell rendering was cut out the middleman a bit on this method. now, instead of the unique `CellColors` interface, we just return `CSSProperties` from this method so we don't have the extra step of mapping from one to the other in both places.